### PR TITLE
Attribute table style

### PR DESCRIPTION
**Pull request by kevinkidwell**
Cleaning up the styles on the information table for index map features. Details:
- Added classes to create structure and make the table elements responsive
- Mimic the layout of Metadata section above map for consistency
- Added spacing to give some visual separation to each heading and information group

### CODE REVIEW

File: app/assets/stylesheets/geoblacklight/modules/index_maps.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.index-map-info {
+	margin-top: 20px;
```
Comment (cappadona) [inline app/assets/stylesheets/geoblacklight/modules/index_maps.scss]:
```suggestion
  margin-top: 20px;
```

Comment (cappadona) [reply app/assets/stylesheets/geoblacklight/modules/index_maps.scss]: tab -> spaces

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <div class="index-map-info">
   {{#if title}}
-    <h3>{{title}}</h3>
+    <div class="col-sm-12">
+      <h3>{{title}}</h3>
+    </div>
   {{/if}}
-  <div class="">
+  <div class="col-sm-12">
```
Comment (cappadona) [inline app/assets/javascripts/geoblacklight/templates/index_map_info.hbs:7]: Noticed that you went with stacking at small breakpoint vs medium breakpoint used by GBL's `document-metadata` dl. Was that by design?

**Review by cappadona — COMMENTED** [review]
Sorry for not getting to this sooner.

I have to admit that while I appreciate the consistency between the document metadata and the feature metadata, this new alignment of the feature metadata feels a bit awkward juxtaposed with the corresponding thumbnail.
> I know neither of us are crazy about the default behavior of rendering the feature meta below the map, so maybe this is a moot point.

screen shot 2019-01-25 at 2 10 55 pm

File: app/assets/javascripts/geoblacklight/templates/index_map_info.hbs (hunk 2 above)
Comment (kevinkidwell) [reply app/assets/javascripts/geoblacklight/templates/index_map_info.hbs:7]: I did this on purpose. I wanted to make sure that it consistently kept the same width/spacing across responsive sizes. I always default to using the smaller breakpoints if it's going to be the same all the way to the smallest screen size.

**Comment by kevinkidwell** [comment]
I don't really enjoy the spacing either but I think it's better like that than it was as the default. This is something that we could always bring up to the larger GBL community. Maybe mention it on the index maps issue in their project?

**Pull request merged**